README.md

```diff
@@ -283,6 +284,14 @@ cleaned up.
 </clean>
 ```
 
+#### Cleaning `catalog_category_product` entity
+A `catalog_category_product` entity isn't supported by Magento and therefore cleaning this entity works different then stated above. 
+`AvS_FastSimpleImport` supports three different types of import behaviours: 
+- Append Complex Data
+- Replace Existing Complex Data (default)
+- Delete Entities
```

Comment:
> A `catalog_category_product` entity isn't supported by Magento and therefore cleaning
Wut? `catalog_category_product` is een tabel in de database waar de relatie tussen een categorie en product wordt gelegd? Je schrijft hier over de import behaviors?